erpnext/accounts/doctype/cashier_closing/cashier_closing.json

```diff
@@ -20,7 +20,440 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
-   "columns": 0, 
+   "columns": 0, {
```

Comment:
Incorrect JSON format. Have you manually modified this file?

Reply:
yes, but i think i fixed it now.

Reply:
Please don't modify it manually. The `modified` timestamp must be auto-generated. You can edit the DocType.

Reply:
ok, i got it but i see there is no conflicts or errors by the json file so, u can merge it ! or i must re fix it ?

Reply:
It must be fixed in order to correctly update database while migration. Please make changes using the doctype. Thanks.

Reply:
ok, i hope its fixed now

Reply:
now i fixed it to be correct with the current version in develop branch + adding  the returns, do i need to make any other changes, or its ready for accept?